MonoGame.Framework.Content.Pipeline/Serialization/Compiler/TextureCubeWriter.cs

```diff
@@ -20,6 +20,8 @@ protected internal override void Write(ContentWriter output, TextureCubeContent
             if (!level0.TryGetFormat(out format))
                 throw new Exception("Couldn't get format for TextureCubeContent.");
 
+            format = value.GetActualFormat(format);
```

Comment:
> Is the texture processing different if the texture is flagged as sRGB? 

No it is not... the texture data is 100% the same.  Pretty much all game textures are gamma corrected and in the non-linear sRGB format.  This is just letting the GPU hardware know this fact so that it can correctly decode it to a linear format before processing.

This is why I originally was looking for a solution that didn't involve adding this to the pipeline.  It doesn't change the pipeline at all really...  it is more of an issue on how the texture is used in your game.